aikau/src/main/resources/alfresco/services/actions/CopyMoveService.js

```diff
@@ -241,14 +241,31 @@ define(["dojo/_base/declare",
        * @param {object} payload
        * @fires module:alfresco/core/topics#DISPLAY_NOTIFICATION
        */
-      onActionSuccess: function alfresco_services_ActionService__onActionSuccess(payload) {
+      onActionSuccess: function alfresco_services_actions_CopyMoveService__onActionSuccess(payload) {
          // jshint unused:false
          var subscriptionHandles = lang.getObject("requestConfig.subscriptionHandles", false, payload);
          if (subscriptionHandles)
          {
             this.alfUnsubscribeSaveHandles(subscriptionHandles);
          }
-         if (payload.response.overallSuccess === true)
+         if (payload.response.successCount === 0)
```

Comment:
Just to double-check; the situation cannot occur where `successCount > 0` but `successCount < payload.response.totalResults`?